src/raftstore/store/worker/pd.rs

```diff
@@ -1118,6 +1119,7 @@ mod tests {
         sum
     }
 
+    #[cfg(not(target_os = "macos"))]
```

Comment:
Doesn't it report warnings on Mac OS because struct `RunnerTest` is not used?

Reply:
It does, but it didn't bother me. I've addressed it in any case.